### PR TITLE
Crashes while using trackAttemptedNetwork and trackUnreachableNetwork

### PR DESCRIPTION
**Pull request by anshumansworld**
This patch includes fix of crash while using `trackAttemptedNetwork` and `trackUnreachableNetwork`, for this a `null` check has been added.

### CODE REVIEW

File: mediation/src/main/java/net/pubnative/mediation/insights/model/PubnativeInsightModel.java [hunk 1]
```diff
@@ -169,8 +169,10 @@ public void trackUnreachableNetwork(PubnativePriorityRuleModel priorityRuleModel
         PubnativeInsightCrashModel crashModel = new PubnativeInsightCrashModel();
         crashModel.error = exception.getMessage();
         crashModel.details = exception.toString();
-        mData.addUnreachableNetwork(priorityRuleModel.network_code);
-        mData.addNetwork(priorityRuleModel, responseTime, crashModel);
+        if (priorityRuleModel != null) {
```
Comment (cerberillo) [inline mediation/src/main/java/net/pubnative/mediation/insights/model/PubnativeInsightModel.java]: you should also check the network code nullity or emptiness

File: mediation/src/main/java/net/pubnative/mediation/insights/model/PubnativeInsightModel.java [hunk 2]
```diff
@@ -185,8 +187,10 @@ public void trackAttemptedNetwork(PubnativePriorityRuleModel priorityRuleModel,
         PubnativeInsightCrashModel crashModel = new PubnativeInsightCrashModel();
         crashModel.error = exception.getMessage();
         crashModel.details = exception.toString();
-        mData.addAttemptedNetwork(priorityRuleModel.network_code);
-        mData.addNetwork(priorityRuleModel, responseTime, crashModel);
+        if (priorityRuleModel != null) {
```
Comment (cerberillo) [inline mediation/src/main/java/net/pubnative/mediation/insights/model/PubnativeInsightModel.java]: same here

File: mediation/src/main/java/net/pubnative/mediation/insights/model/PubnativeInsightModel.java [hunk 3]
```diff
@@ -197,8 +201,10 @@ public void trackAttemptedNetwork(PubnativePriorityRuleModel priorityRuleModel,
     public void trackSuccededNetwork(PubnativePriorityRuleModel priorityRuleModel, long responseTime) {
 
         Log.v(TAG, "trackSuccededNetwork");
-        mData.network = priorityRuleModel.network_code;
-        mData.addNetwork(priorityRuleModel, responseTime, null);
+        if (priorityRuleModel != null) {
```
Comment (cerberillo) [inline mediation/src/main/java/net/pubnative/mediation/insights/model/PubnativeInsightModel.java]: same here

**Comment by psychocandy** [comment]
[(0.5) Protect for crashes while using trackAttemptedNetwork and trackUnreachableNetwork [0.5]](https://trello.com/c/uTo0PDE7/1328-0-5-protect-for-crashes-while-using-trackattemptednetwork-and-trackunreachablenetwork)

**Pull request merged**